docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md

```diff
@@ -0,0 +1,140 @@
+---
+title:  Hue 4.11 and its new dialects and features are out!
+author: Hue Team
+type: post
+date: 2023-01-23T00:00:00+00:00
+url: /blog/2023-01-23-hue-4.11-hplsql-sparksql-iceberg/
+sf_thumbnail_type:
+  - none
+sf_thumbnail_link_type:
+  - link_to_post
+sf_detail_type:
+  - none
+sf_page_title:
+  - 1
+sf_page_title_style:
+  - standard
+sf_no_breadcrumbs:
+  - 1
+sf_page_title_bg:
+  - none
+sf_page_title_text_style:
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.11
+  - Release
+
+---
+
+Hi Data Explorers,
+
+The Hue Team is glad to thanks all the contributors and release Hue 4.11!
+
+<a href="https://cdn.gethue.com/uploads/2021/02/hue-4.9.png">
+  <img src="https://cdn.gethue.com/uploads/2021/02/hue-4.9.png" />
+</a>
+
+&nbsp;
+
+**Note: This is the last Hue release supporting Python2.7. From future releases, Hue will support Python3.8+**
+
+
+Here is a summary of the [main improvements](/categories/version-4.11/) of 4.11 on top of the previous [4.10](/blog/hue-4-10-sql-scratchpad-component-rest-api-small-file-importer-slack-app/) release:
```

Comment:
Here's a summary of [what's new](/categories/version-4.11/) in Hue 4.11:

(Please remove "on top of the previous [4.10](/blog/hue-4-10-sql-scratchpad-component-rest-api-small-file-importer-slack-app/) release" and add the link to 4.10 at the bottom of the page)